srv/fluffi/data/fluffiweb/app/constants.py

```diff
@@ -50,4 +51,8 @@ def getConfigJson():
 FTP_CONNECTOR = FTPConnector(config.FTP_URL)
 ANSIBLE_REST_CONNECTOR = AnsibleRESTConnector("http://pole.fluffi:8888/api/v2/", "admin", "admin")
 
+SYNC = True
+SYNCING_FILEPATH = os.getcwd() + "/syncing.lock"
```

Comment:
just a thought, I have not checked: what is the working directory here? is this outside of any persistent paths in the container? otherwise this would deadlock if we restart the container during the syncing phase, because the lockfiles survive a container restart

Reply:
The lock files (also for the archive download) are currently saved in the persistent part. I will remove the "os.getcwd" then it will be in the root of the Docker container (not persistent).